tests/quantization/autoawq/test_awq.py

```diff
@@ -104,15 +107,41 @@ def test_quantized_model_conversion(self):
         """
         from awq.modules.linear import WQLinear_GEMM, WQLinear_GEMV
 
-        self.tokenizer(self.input_text, return_tensors="pt").to(torch_device)
+        from transformers.integrations.awq import replace_with_awq_linear
+
+        model_id = "facebook/opt-350m"
```

Comment:
Why a checkpoint (we don't need trained weights here) and why this checkpoint when a [smaller one exists](https://huggingface.co/facebook/opt-125m/tree/main)? 

Even though `init_empty_weights` is used here - this is just handling loading into ram, it still requires the checkpoint to be downloaded --> larger models make things slower. 

If you replace this with a hf-internal-testing tiny model OR a v. small model defined with a model config this will be more lightweight. In this case we might even be able to remove the accelerate dependency.

Reply:
For tiny model testing I use this - a 68M Llama model: https://huggingface.co/JackFram/llama-68m

Reply:
@amyeroberts thanks! 
the init_empty weights context manager will create a model on the meta device, it will cost 0 RAM regardless of the architecture. during the entire test I am only dealing with the model that is loaded on the meta device and I am not downloading any checkpoint. 

> it still requires the checkpoint to be downloaded --> larger models make things slower.

The only thing that I download is the config file, no model weight is being added here. The accelerate dependency is fine since we use it for all quantization integrations + always downloaded in our docker images for slow tests

Reply:
My bad - I misread and thought `from_pretrained` was being used in the model creation i.e. `model = XxxForCausalLM.from_pretrained`